EpikCord/ext/tasks.py

```diff
@@ -0,0 +1,82 @@
+import datetime
+from typing import Callable, Optional
+import asyncio
+from ..exceptions import TaskFailedError
+
+_EPOCH = datetime.datetime(1970, 1, 1)
+__all__ = (
+    "TimeParser",
+    "Tasks",
+)
+
+
+
+
+@staticmethod
+def total_seconds(
+    day: int = 0,
+    hour: int = 0,
+    minute: int = 0,
+    seconds: float = 0,
+) -> float:
+    hour += day * 24
+    minute += hour * 60
+    seconds += minute * 60
+    return seconds
+
+
+class Tasks:
+    def __init__(self, client) -> None:
+        self.client = client
+
+    def add_task(
```

Comment:
Where is `limit`? It's in the docstring, but not in the signature for this function

Reply:
limit is a kwarg 4 your information

Reply:
Make it explicit. If not then it's going to be passed into task callbacks

Reply:
i did

Reply:
I made it explicit P.F.A latest commit